ros2topic/ros2topic/verb/echo.py

```diff
@@ -112,15 +118,15 @@ def subscriber(node, topic_name, message_type, callback):
         rclpy.spin_once(node)
 
 
-def subscriber_cb(truncate_length):
+def subscriber_cb(truncate_length, noarr, nostr):
     def cb(msg):
-        nonlocal truncate_length
-        print(message_to_yaml(msg, truncate_length), end='---\n')
+        nonlocal truncate_length, noarr, nostr
+        print(message_to_yaml(msg, truncate_length, noarr, nostr), end='---\n')
```

Comment:
Please pass keyword arguments explicitly with the keyword.

Same below.

Reply:
@jacobperron Friendly ping.

Reply:
https://github.com/ros2/ros2cli/pull/317